book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -135,17 +135,17 @@ $ git commit -a -m 'Finish the new footer [issue 53]'
 1 file changed, 1 insertion(+)
 ----
 
-.Arbeiten an `iss53` weitergeführt
-image::images/basic-branching-6.png[Arbeiten an `iss53` weitergeführt]
+.Arbeiten an `iss53` fortsetzen
+image::images/basic-branching-6.png[Arbeiten an `iss53` fortsetzen]
 
 Es ist erwähnenswert, dass die Arbeit, die Sie in Ihrem `hotfix`-Branch durchgeführt haben, nicht in den Dateien in Ihrem `iss53`-Branch enthalten ist.
-Wenn Sie diese Änderungen übernehmen müssen, können Sie in Ihren `master`-Branch den `iss53`-Zweig einbinden indem Sie `git merge master` ausführen, oder Sie können warten, bis Sie sich später entscheiden, den `iss53`-Zweig wieder zurück nach `master` zu pullen.
+Wenn Sie diese Änderungen übernehmen müssen, können Sie in Ihren `master`-Branch den `iss53`-Zweig einbinden indem Sie `git merge master` ausführen, oder Sie können warten, bis Sie sich später entscheiden, den `iss53`-Zweig wieder zurück nach `master` zu übernehmen (engl. pullen).
 
 [[_basic_merging]]
 ==== Einfaches Merging
 
 (((Branches, mergen)))(((Merging)))
-Angenommen, Sie haben entschieden, dass Ihr Issue #53 abgeschlossen ist und Sie bereit sind, ihn in Ihren „Master“ Branch zu integrieren.
+Angenommen, Sie haben entschieden, dass Ihr Issue #53 abgeschlossen ist und Sie bereit sind, ihn in Ihren `master`-Branch zu integrieren.
```

Comment:
```suggestion
Angenommen, Sie haben entschieden, dass Ihr Issue #53 abgeschlossen ist und Sie bereit sind, ihn in Ihren Branch `master` zu integrieren.
```

Reply:
Ich sehe es ein, dass ein einzelner Dash nach einem als Code formatierten Text optisch schlecht aussieht. Hier ist dies nicht der Fall und es gibt zahlreiche Fälle, wo Text schon so formatiert ist, was auch völlig in Ordnung ist, wie ich finde.
"Deine" Variante gibt es auch häufiger im Buch und liest sich jedesmal komisch.